app/models/manageiq/providers/openstack/cloud_manager/host_aggregate/operations.rb

```diff
@@ -0,0 +1,37 @@
+module ManageIQ::Providers::Openstack::CloudManager::HostAggregate::Operations
```

Comment:
TBH you dont have to follow this convention of moving stuff into modules.
This was done to eventually move those methods to actual classes.
So, in this case, if you cannot make those operations a class of its own, which seems unlikely to me, just move the methods to `HostAggregate`